src/main/java/Duke.java

```diff
@@ -1,10 +1,245 @@
+import java.util.*;
+import java.io.File;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.time.*;
+import java.time.format.DateTimeFormatter;
+import java.text.SimpleDateFormat;
```

Comment:
Consider including a line spacing between different import groups to make it look clearer